x86 Linux build fails

x86 Linux build fails with message:
```
  /home/tmustafin/dotnet/runtime/src/coreclr/vm/i386/jithelp.S:314:21: error: cannot use more than one symbol in memory operand
      add eax, offset _GLOBAL_OFFSET_TABLE_+1
                      ^
  vm/wks/CMakeFiles/cee_wks_core.dir/build.make:3330: recipe for target 'vm/wks/CMakeFiles/cee_wks_core.dir/__/i386/jithelp.S.o' failed
```
Problem occures on https://github.com/dotnet/runtime/blob/738b09bee351a60f1a7c037d5968b1e4145f6c82/src/coreclr/vm/i386/jithelp.S#L395.
Build command:
```
sudo ./eng/common/cross/build-rootfs.sh x86 bionic
ROOTFS_DIR=`pwd`/.tools/rootfs/x86 ./build.sh --cross --clang9 --arch x86 --configuration Release --subset clr.runtime+clr.corelib+clr.iltools+clr.dactools+clr.jit /p:EnableSourceLink=false
```

cc @janvorli @jkotas @alpencolt @gbalykov

--- a/src/coreclr/vm/i386/jithelp.S
+++ b/src/coreclr/vm/i386/jithelp.S
@@ -392,7 +392,9 @@ LEAF_ENTRY JIT_WriteBarrier_Callable, _TEXT
 1:
     pop     eax
 2:
-    add     eax, offset _GLOBAL_OFFSET_TABLE_+1 // (2b - 1b)
+.att_syntax
+    addl     $_GLOBAL_OFFSET_TABLE_+(2b-1b), %eax
+.intel_syntax noprefix
     mov     eax, dword ptr [eax + C_FUNC(JIT_WriteBarrierEAX_Loc)@GOT]
     xchg    eax, dword ptr [esp]
     ret

